policy/modules/kernel/kernel.if

```diff
@@ -380,7 +380,7 @@ interface(`kernel_rw_netlink_audit_sockets',`
 		type kernel_t;
 	')
 
-	allow $1 kernel_t:netlink_audit_socket { rw_netlink_socket_perms };
+	allow $1 kernel_t:netlink_audit_socket { create_netlink_socket_perms };
 ')
```

Comment:
Since this is explicitly `rw`, I think we shouldn't change this. Since it's not a `self` permission and no setsockcreate, the create wouldn't be used.